CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### Deprecated
 ### Removed
 ### Fixed
+- Fixed issue 769: Collapse not preserved when chaining a search instance ([#771](https://github.com/opensearch-project/opensearch-py/pull/771))
```

Comment:
@imvtsl, thank you for the contribution. Please change the changelog entry to something like: 'Fixed Search helper to ensure proper retention of the _collapse attribute in chained operations.' No need to mention the issue number.

Reply:
I made the change. Thank you!
I believe, you will have to approve again. It dismissed review automatically as soon as I pushed the change.